### PR TITLE
GH(GH-3546):Prepare video for Adaptive UI Layout in Blazor DataGrid Component

### PR DESCRIPTION
**Pull request by ghost**
Description:
Add a sample page for adaptive UI layout with SfMediaQuery

### CODE REVIEW

File: Grid-AdaptiveLayout/Grid-AdaptiveLayout/Pages/_Layout.cshtml [hunk 1]
```diff
@@ -11,8 +11,8 @@
     <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
     <link href="css/site.css" rel="stylesheet" />
     <link href="Grid-AdaptiveLayout.styles.css" rel="stylesheet" />
-     <link href="https://cdn.syncfusion.com/blazor/20.2.38/styles/fluent.css" rel="stylesheet" />
-    <script src="https://cdn.syncfusion.com/blazor/20.2.38/syncfusion-blazor.min.js" type="text/javascript"></script>
+     <link href="https://cdn.syncfusion.com/blazor/20.4.48/styles/fluent.css" rel="stylesheet" />
```
Comment (syncsiva) [inline Grid-AdaptiveLayout/Grid-AdaptiveLayout/Pages/_Layout.cshtml:14]: refer from static web assets

File: Grid-AdaptiveLayout/Grid-AdaptiveLayout/Shared/NavMenu.razor [hunk 2]
```diff
@@ -16,7 +16,7 @@
         </div>
         <div class="nav-item px-3">
             <NavLink class="nav-link" href="counter">
-                <span class="oi oi-plus" aria-hidden="true"></span> Counter
+                <span class="oi oi-plus" aria-hidden="true"></span> AdaptiveUI with SfMediaQuery
```
Comment (syncsiva) [inline Grid-AdaptiveLayout/Grid-AdaptiveLayout/Shared/NavMenu.razor:19]: DataGrid Adaptive UI using Media Query

File: Grid-AdaptiveLayout/Grid-AdaptiveLayout/Pages/Counter.razor [hunk 3]
```diff
@@ -1,18 +1,51 @@
 ﻿@page "/counter"
```
Comment (syncsiva) [inline Grid-AdaptiveLayout/Grid-AdaptiveLayout/Pages/Counter.razor:2]: add intro title in page as

Enable/Disable DataGrid Adaptive UI using Media Query Component

**Review by syncsiva — APPROVED** [review]

File: Grid-AdaptiveLayout/Grid-AdaptiveLayout/Pages/_Layout.cshtml (hunk 1 above)
Comment (ghost) [reply Grid-AdaptiveLayout/Grid-AdaptiveLayout/Pages/_Layout.cshtml:14]: Referred from static web assets 
https://github.com/SyncfusionExamples/blazor-datagrid-render-adaptive-layout/commit/9b4416874df66fe6c2afe45f1b0933f7d4b188f7

File: Grid-AdaptiveLayout/Grid-AdaptiveLayout/Shared/NavMenu.razor (hunk 2 above)
Comment (ghost) [reply Grid-AdaptiveLayout/Grid-AdaptiveLayout/Shared/NavMenu.razor:19]: Changed the side bar name as DataGrid Adaptive UI using Media Query
https://github.com/SyncfusionExamples/blazor-datagrid-render-adaptive-layout/commit/9b4416874df66fe6c2afe45f1b0933f7d4b188f7

**Pull request merged**